ui/ducks/swaps/swaps.js

```diff
@@ -395,15 +404,22 @@ export const fetchQuotesAndSetQuoteState = (
   return async (dispatch, getState) => {
     const state = getState();
     const chainId = getCurrentChainId(state);
-    let swapsFeatureIsLive = false;
+    let swapsLivenessForNetwork = {
+      swapsFeatureIsLive: false,
+      useNewSwapsApi: false,
+    };
     try {
-      swapsFeatureIsLive = await fetchSwapsFeatureLiveness(chainId);
+      const swapsFeatureFlags = await fetchSwapsFeatureFlags();
+      swapsLivenessForNetwork = getSwapsLivenessForNetwork(
+        swapsFeatureFlags,
+        getCurrentChainId(getState()),
```

Comment:
Nit: `getState()` is called above when the `state` const is set. The `state` const could be used here instead of called `getState()` again.

Reply:
Actually, the same is true for `chainId` and `getCurrentChainId`. So instead of `getCurrentChainId(getState())` here, it can just be `chainId`